docs/additional/interesting_aspects_en.md

```diff
@@ -0,0 +1,249 @@
+We recommend that a local development environment is defined in the form of a `docker-compose.yml` file and can be easily started at any time with `docker-compose up` and shut down with `docker-compose down`.  
+For an example [see CAS][local-cas-example].
+
+[local-cas-example]: https://github.com/cloudogu/cas/blob/develop/app/docker-compose.yml
+
+## Quality Assurance for Dogus
+
+### Container Validation (GOSS)
+
+Container validation assures the correct configuration of the container, which is necessary for the smooth execution of the software.  
+
+For this purpose, we use the server validation program [goss][goss] in our Dogus.  
+The actual goss tests are created in the following data structure in the dogu:
+- `<root directory of the Dogu>/`
+  - `spec/`
+    - `goss/`
+      - `goss.yml` (contains all goss tests)
+
+The goss tests should ideally check all important aspects for the smooth start of a dogu.  
+This includes aspects like:
+- Checking permission for scripts (startup.sh), volumes.
+- Checking the UID & GID for files (startup.sh, resources, volumes, write-dependent paths)
+- Accessibility of the software: TCP health checks
+
+For pipeline integration, we recommend using the `ecoSystem.verify()` method from the [dogu-build-lib][dogu-build-lib].  
+An example call looks like the following in a Jenkinsfile:
+
+```groovy
+stage('Verify') {
+    ecoSystem.verify("/dogu")
+}
+```
+
+This internally calls the `cesapp verify` command, which runs the goss tests.
+
+[goss]: https://github.com/goss-org/goss
+[dogu-build-lib]: https://github.com/cloudogu/dogu-build-lib
+
+## Documentation
+
+Every tool should have a reasonable documentation about used features, which supports usage or support.
+To allow easy integration into our existing documentation infrastructure, the documentation should be placed in a `/docs` folder located in the repository root.
+
+By default, the docs include the following:
+- Features of the Dogu
+- Installation of the Dogu
+- Configuration of the Dogu
+- Development on the Dogu
+- System requirements of the Dogu (nice to have)
+- Readme
+- Changelog
+
+### License
+
+The license for using the Dogu should be clearly stated.
+This can be specified in the Readme, for example.
+
+### Readme
+
+A Readme is useful both for your own developments and in a partner context with external developers.  
+In the case of public repositories, however, a higher standard should be applied to a Readme.
+
+A Readme file is located in the repository root and concisely describes notes on the general purpose of the Dogu.
+This can be in the form of a quickstart guide, for example, describing how to set up a local development environment.
+It should also include a reference to the documentation folder and can also refer to specific pages of the documentation.
+Finally, it is recommended to specify required resources for use and a person responsible for the Dogu.
```

Comment:
This sentences mixes two things which should be separated:

1. describing the purpose of the dogu (aimed at dogu users)
2. a development quickstart (aimed at dogu developers)

I suggest to make the possible uses of a Readme more plastic, like so:

```suggestion
A Readme file is located in the repository root and concisely describes notes on the general purpose of the Dogu. For users this can be some sorts of quickstart guide of how to use the Dogu, or even a feature list. For Dogu developers the readme may provide insights of how collaboration is possible or which copyright license was chosen. Additional docs can be linked to provide further information for both users and developers.

Administrators may want to find information about the resource consumption of this dogu. Finally, the readme should clarify who is responsible for this Dogu.
```